x-pack/plugins/security_solution/public/management/pages/policy/store/policy_details/reducer/initial_policy_details_state.ts

```diff
@@ -25,6 +25,8 @@ export const initialPolicyDetailsState: () => Immutable<PolicyDetailsState> = ()
     online: 0,
     total: 0,
     other: 0,
+    all: 0,
```

Comment:
Got a question, what about all the places where we already use the `.total`? Should we change all the `.total` references by `.all`? Or, is `.total` still working as before?

Reply:
@dasansol92 Yeah `.total` should work fine as it is but it is a good idea to replace `.total` usages with `.active`.

Reply:
yes, `total` still works as it is only deprecated, it should be changed eventually by the deprecation is removed (next major)